Cinema/src/main/java/com/example/Cinema/model/ProjectionsUpdate.java

```diff
@@ -9,19 +12,27 @@
 public class ProjectionsUpdate {
```

Comment:
Same as for Projections class. This represents a single Projection update...

Reply:
Fixed